packages/mint-components/src/components/sqm-portal-container/sqm-portal-container-view.tsx

```diff
@@ -16,25 +25,29 @@ export function PortalContainerView(
 ) {
   const style = {
     Container: {
-      display: "grid",
+      display: props.display || "grid",
```

Comment:
General comment I have seen this in other places, I dont really think we need to have a fallback in the code if the prop has a default value in the controller

Reply:
I believe this was a new prop to allow `display: flex`, and for backwards compatibility, we had made it optional and made the default fallback to `grid` as in the original